common/traits/000_wc_leadership_traits.txt

```diff
@@ -10,22 +10,26 @@ archmagus_leader = {
 	customizer = no
 
 	command_modifier = {
+		speed = 0.05	#Time magic
```

Comment:
A small supply limit bonus would be better (conjures food?)
Units already have a high speed in the mod, which makes all speed modifiers useless